styles/Home.module.css

```diff
@@ -70,6 +70,18 @@
   margin-top: 20px;
 }
 
+.trade {
+  width: 100%;
+  text-align: center;        
+  margin: 0;
+  padding: 1rem 1rem;
```

Comment:
Since both the values are same, we could use `padding: 1rem` directly
Source: https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties

![image](https://user-images.githubusercontent.com/56217868/118002569-5ee3fa80-b365-11eb-86c4-5675a61a59dd.png)

Reply:
yeah right! 👍

Reply:
Changes done!